src/experiments/tests/gpid-prebid.ts

```diff
@@ -0,0 +1,29 @@
+import type { ABTest } from '@guardian/ab-core';
+
+export const gpidPrebidAdUnits: ABTest = {
+	id: 'gpidPrebidAdUnits',
```

Comment:
```suggestion
	id: 'GpidPrebidAdUnits',
```
I think this ID needs to be capitalised to work

Reply:
great spot thanks @emma-imber 👍